test/test_validation.py

```diff
@@ -463,6 +463,30 @@ def test_invalid_items(self):
             'Incorrectly accepts integers outside of the range'
         )
 
+   def test_valid_character_items(self):
+        self.validate_and_compare(
+            [
+                7,
+                "8",
+                8
+            ],
+            True,
+            "Does not accept integer in a string"
+        )
+
+    # TODO: I was trying to validate [7, "eight", 8], but the test failed.
+    # it seems that the test check for all the elements as False, and the first element is true.
+    def test_invalid_character_items(self):
```

Comment:
This is good. This satisfies me that it won't throw on error when encountering non-numerics.